tests/wpunit/Players_Controller_Test.php

```diff
@@ -0,0 +1,49 @@
+<?php
```

Comment:
This is my first attempt at writing some unit tests for the plugin. I will probably add more tests to this Test suite, and then follow it up with some Integration tests and Functional tests